graph/src/components/store.rs

```diff
@@ -393,19 +415,255 @@ pub trait Store: Send + Sync + 'static {
 
     /// Counts the total number of entities in a subgraph.
     fn count_entities(&self, subgraph: SubgraphDeploymentId) -> Result<u64, Error>;
-}
 
-pub trait SubgraphDeploymentStore: Send + Sync + 'static {
     fn resolve_subgraph_name_to_id(
```

Comment:
Note: this method is not new and has not actually changed -- I just moved the body to here from the diesel/postgres store